src/initialize.js

```diff
@@ -1,4 +1,5 @@
 import 'styles/index';
+import 'babel-polyfill';
```

Comment:
This works in conjunction with the babel-preset-env plugin -- it is not really importing the entire babel-polyfill collection, just those needed.